etcdmain/gateway.go

```diff
@@ -77,6 +78,23 @@ func newGatewayStartCommand() *cobra.Command {
 	return &cmd
 }
 
+func stripSchema(eps []string) []string {
```

Comment:
Can we add unit tests for this function? I would like to increase the etcd coverage.